packages/grid/x-data-grid-pro/src/index.ts

```diff
@@ -38,3 +38,8 @@ export type {
   GridColumns,
   GridComparatorFn,
 } from './typeOverload';
+export type {
```

Comment:
To improve after #3965 to remove the explicit export here

Reply:
Do you still see space for improvement?

Reply:
What is the goal of this export ?

Shouldn't it be exported b one of the other `export *` from this file ?